libs/baf-contract/contract/src/lib.rs

```diff
@@ -0,0 +1,116 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::env::keccak256;
+use near_sdk::AccountId;
+use near_sdk::{collections::UnorderedMap, env, near_bindgen};
+use ed25519_dalek::Verifier;
+use std::convert::TryInto;
+
+#[global_allocator]
+static ALLOC: near_sdk::wee_alloc::WeeAlloc = near_sdk::wee_alloc::WeeAlloc::INIT;
+
+type EdPK = [u8; 64];
+type SecpPK = [u8; 65];
+
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct AccountInfo {
+    ed_pk: EdPK,
+    accountId: AccountId,
+}
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct BafContract {
+    owner_id: AccountId,
```

Comment:
if this is the owner of the contract, we can save contract storage and use [`current_account_id`](
https://docs.rs/near-sdk/3.1.0/near_sdk/env/fn.current_account_id.html) instead.

Reply:
Oh for sure